src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -34,11 +34,16 @@ public void execute() {
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
+
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
+    if(m_ShooterSubsystem.wheelReady()){
+      m_ShooterSubsystem.shoot();
+      return true;
+    }
```

Comment:
Nothing functionally wrong here. but isFinished() really shouldn't do anything other than return whether the command is done. Put shoot() in the end() method, and check interrupted there to make sure the flywheel is actually ready and the command didn't get canceled for some other reason.